python/py-pysap/Portfile

```diff
@@ -30,5 +30,5 @@ if {${name} ne ${subport}} {
 
     depends_lib-append \
                     port:py${python.version}-cryptography \
-                    port:py${python.version}-scapy
+                    bin:scapy-${python.branch}:py${python.version}-scapy
```

Comment:
are you sure that you want to use the `bin` dependency here instead of the `path`? This would allow stuff installed in `/usr/bin`, `/bin/` to allows satisfy the dependency. I assume you meant to have a MacPorts version here?

Reply:
Good point. I thougth it checks only `${prefix}/bin`.

Reply:
https://guide.macports.org/chunked/reference.dependencies.html#reference.dependencies.types